docs/en/ingest-management/create-agent-policies-no-UI.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[create-a-policy-no-ui]]
```

Comment:
My guess is that Asciidoctor includes are case sensitive. You're including `create-agent-policies-no-ui.asciidoc`, but the file is named `create-agent-policies-no-UI.asciidoc`. Lowercase the `ui` in the filename and that might fix it.

Reply:
:scream: That must be it!

Reply:
Thanks, :eagle: :eye: Morelli